sdk/src/androidTest/java/com/mapbox/maps/OfflineTest.kt

```diff
@@ -419,9 +422,11 @@ class OfflineTest {
     }
     switchAirplaneMode()
     verifyAirplaneModeEnabled()
+    // Show map with the zoom level larger than download range to trigger loading error
     prepareMapView(
       observer,
-      listOf(MapEvents.MAP_IDLE, MapEvents.MAP_LOADING_ERROR)
+      listOf(MapEvents.MAP_IDLE, MapEvents.MAP_LOADING_ERROR),
+      ZOOM + 3
```

Comment:
Why is it 3 here? 1 in some places and 2 in some other places.
Perhaps we could use one number and move it to const.

Reply:
The download rang is [zoom - 2, zoom + 2], use zoom + 3 here to trigger loading error.